src/WebJobs.Script.WebHost/Diagnostics/MetricsEventManager.cs

```diff
@@ -133,6 +161,44 @@ private static void WriteFunctionsMetricEvent(string executionId, ulong executio
                 MetricEventSource.Log.RaiseFunctionsMetricEvent(executionId, executionTimeSpan, executionCount, executionStage);
             }
 
+            private void RaiseMonitoringEvent()
+            {
+                List<FunctionMonitoringMetrics> monitoringEventsList = GetMonitoringQueueSnapshot();
+                
+                var aggregatedEventsPerFunction = from item in monitoringEventsList
+                                                  group item by new { item.FunctionName } into FunctionGroups
+                                                  select new
+                                                  {
+                                                      FunctionName = FunctionGroups.Key.FunctionName,
+                                                      StartedCount = Convert.ToUInt64(FunctionGroups.Count(x => x.ExecutionStage == ExecutionStage.Started)),
+                                                      FailedCount = Convert.ToUInt64(FunctionGroups.Count(x => x.ExecutionStage == ExecutionStage.Failed)),
+                                                      SucceddedCount = Convert.ToUInt64(FunctionGroups.Count(x => x.ExecutionStage == ExecutionStage.Succedded)),
+                                                      TotalExectionTimeInMs = Convert.ToUInt64(FunctionGroups.Sum(x => Convert.ToDecimal(x.ExecutionTimeInMs)))
+                                                  };
+
+                var siteName = Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME");
```

Comment:
Please rebase to master. Ashish is also accessing this, so it should be grabbed once on startup in the constructor